client/src/App.jsx

```diff
@@ -1,24 +1,13 @@
 import React from "react";
-import logo from "./logo.svg";
+import HomeView from "./pages/HomeView";
+import Navbar from "./components/Navbar";
```

Comment:
Use `./components/index.js` for exporting default components from `./components`.

In `./components/index.js` you would have `import Navbar from "./Navbar"` then `export { Navbar }`

Now we can replace the above `import` with `import { Navbar } from "./components"`